grpc/shared/bech32m.go

```diff
@@ -0,0 +1,92 @@
+package shared
+
+// Copied from astria-cli-go bech32m module (https://github.com/astriaorg/astria-cli-go/blob/d5ef82f718325b2907634c108d42b503211c20e6/modules/bech32m/bech32m.go#L1)
+// TODO: organize the bech32m usage throughout the codebase
+
+import (
+	"crypto/ed25519"
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/btcsuite/btcd/btcutil/bech32"
+)
+
+type Address struct {
```

Comment:
add a comment saying these represent astria (?) addresses

Reply:
addressed in https://github.com/astriaorg/flame/pull/30/commits/defc35bb0c32331a823a635dccdb28c7e62d519b